src/main/resources/add-ons-to-index.json

```diff
@@ -783,6 +783,41 @@
         "package": "clinicalsummary"
       }
     },
+    {
+      "uid": "org.openmrs.module.commonlabtest",
+      "type": "OMOD",
+      "name": "Common Lab Test",
+      "maintainers": [
+        {
+          "name": "Owais Hussain"
+        },
+        {
+          "name": "Tahira Niazi"
+        }
+      ],
+      "links": [
+        {
+          "rel": "documentation",
+          "href": "https://wiki.openmrs.org/display/docs/Common+Lab+Module",
+          "title": "Wiki Page"
+        },
+        {
+          "rel": "documentation",
+          "href": "https://wiki.openmrs.org/display/docs/Common+Lab+Module+Rest+API",
+          "title": "REST API"
+        },
+        {
+          "rel": "source",
+          "href": "https://github.com/openmrs/openmrs-module-commonlabtest"
+        }
+      ],
+      "backend": "org.openmrs.addonindex.backend.Bintray",
+      "bintrayPackageDetails": {
+        "owner": "seekme94",
```

Comment:
TLDR: Kindly change the owner to openmrs (I am aware that your module is not in the openmrs account)

We only have a certain quota for using the bintray api to fetch module details from external accounts(non-openmrs bintray accounts). Hence, to not exhaust the quota, I will be linking to your module from the OpenMRS Bintray account. Kindly check [this](https://talk.openmrs.org/t/clarification-needed-on-adding-modules-to-bintray/18613/3) discussion for further context